simulators/utils.py

```diff
@@ -42,13 +65,15 @@ def mjd():
         yearp = year
         monthp = month
 
-    # this checks where we are in relation to October 15, 1582, the beginning
+    # This checks where we are in relation to October 15, 1582, the beginning
```

Comment:
You can use the imperative for the comments, as for the first sentence in the docstring:

`Check where...`